### ISSUE TITLE
list shows archived pipelines

### ISSUE THREAD

**Issue by MaxUlysse** [nf-core/tools#639]
Archived pipeline are listed with `nf-core list`

```console
$ nf-core list
...
nf-core/exoseq             dev        -             -              -
```

**Comment by ewels** [comment]
Suggestion / plan:

* Don't show archived by default
* Add command line option to show archived - format as dim and at the bottom of the default sorting

**Comment by ewels** [comment]
Fixed in #671. If shown, now appear at the bottom of the list and all in black. Version is shown as `archived` instead of `dev` / latest release.

![image](https://user-images.githubusercontent.com/465550/87394473-20987600-c5b0-11ea-8992-e5a2dfcaeca7.png)
